### DISCUSSION TITLE
Sensitivity of the rotary knob

### DISCUSSION THREAD

**Discussion by sou7611** [probonopd/MiniDexed#241] [Q&A] [answered]
How sensitive is the knob supposed to be, and is this adjustable?

I'm finding single clicks often do not register, and I have to twirl the knob a bit.

**Comment by craigyjp** [comment, marked as answer]
Have you added 10nf caps across the encoder pins to Gnd, it certainly improves the encoder action. I added a cap to ground on encoder A, B and switch pins.

**Reply by Banana71** [reply]
Here is the link to fix the problem:
https://github.com/probonopd/MiniDexed/discussions/178#discussion-4035975

**Comment by sou7611** [comment]
Ohhh, neat.  Looks like an easy fix.

**Comment by sou7611** [comment]
Ok, so 10n caps have not fixed the issue.

What I'm getting now on the rotary encoder, with the caps installed, is every SECOND click registers.
